engine/source/systems/physics_system.cpp

```diff
@@ -1,31 +1,46 @@
 ﻿#include "systems/physics_system.hpp"
+#include "components/name_component.hpp"
 #include "components/rigidbody_component.hpp"
 #include "ecs.hpp"
 #include "imgui/imgui.h"
-#include "modules/physics_module.hpp"
+#include "physics_module.hpp"
 
 PhysicsSystem::PhysicsSystem(ECS& ecs, PhysicsModule& physicsModule)
```

Comment:
IMO, I think the PhysicsSystem should also be a part of the physics module (not sure how this would integrate with the OldEngine)